bin/fastserve

```diff
@@ -32,8 +32,6 @@ var OPTS = {
     '-p,--port': 'fast RPC listening port(default: 2030)',
     '-m,--mon-port': 'monitoring server listening port (default: port+800)',
     '-q,--quiesce': 'enable quiesce (default: false)',
```

Comment:
```suggestion
    '-q,--quiesce': 'enable quiesce (default: false)'
```

for 'make check'

Reply:
I don't grok why the UI here is showing that I changed the leading whitespace. I don't think I did.